test/meson.build

```diff
@@ -42,11 +42,11 @@ endif
 
 nnstreamer_test_dep = dependency('nnstreamer-test-devel', required: false)
 
-if get_option('enable-nnstreamer-tensor-filter') and nnstreamer_test_dep.found()
+if get_option('enable-nnstreamer-tensor-filter').enabled() and nnstreamer_test_dep.found()
```

Comment:
you might need the more test cases to evaluate nnstreamer-tensor-trainer.

Reply:
Okay, thank you.